### PR TITLE
chore: update hcsshim for NPM

### PR DESCRIPTION
**Pull request by matmerr**
**Reason for Change**:

Needed for latest hcsshim update

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

File: cni/network/network_windows.go [hunk 1]
```diff
@@ -14,8 +14,8 @@ import (
 	"github.com/Azure/azure-container-networking/log"
 	"github.com/Azure/azure-container-networking/network"
 	"github.com/Azure/azure-container-networking/network/policy"
-	"github.com/Microsoft/hcsshim"
-	hnsv2 "github.com/Microsoft/hcsshim/hcn"
+	"github.com/microsoft/hcsshim"
```
Comment (vakalapa) [inline cni/network/network_windows.go:17]: I think it is "M" not microsoft with lowercase. Lets correct that, once i made that change it helped in my local repo.

**Comment by vakalapa** [comment]
@matmerr  Please rebase this when you get a chance.

File: go.mod [hunk 2]
```diff
@@ -5,17 +5,15 @@ go 1.16
 require (
 	code.cloudfoundry.org/clock v1.0.0 // indirect
 	github.com/Masterminds/semver v1.5.0
-	github.com/Microsoft/go-winio v0.4.15-0.20190919025122-fc70bd9a86b5
-	github.com/Microsoft/hcsshim v0.8.10-0.20200506181021-222e9efadbe0
+	github.com/Microsoft/go-winio v0.4.17
+	github.com/Microsoft/hcsshim v0.8.18
```
Comment (davinci26) [inline go.mod:9]: This version works for #1003 but there is a newer version available

**Comment by davinci26** [comment]
@matmerr  @ashvindeodhar this PR is needed before we can merge #1003

Is this stale? If there are any outstanding comments and @matmerr is busy I can take over as well

**Review by rbtr — APPROVED** [review]

**Pull request merged**